src/GitHub.VisualStudio/GitHub.VisualStudio.csproj

```diff
@@ -11,6 +11,7 @@
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
     <TargetFrameworkProfile />
+    <ApplicationVersion>2.3.1.0</ApplicationVersion>
```

Comment:
This change shouldn't be in here. Bad merge somewhere?

Reply:
Yep, bad merge - well spotted!